streams/src/test/java/org/apache/kafka/streams/kstream/internals/SessionTupleForwarderTest.java

```diff
@@ -80,9 +80,10 @@ private void shouldForwardRecordsIfWrappedStateStoreDoesNotCache(final boolean s
         new SessionTupleForwarder<>(store, context, null,
             sendOldValued)
             .maybeForward(
-                new Windowed<>("key", new SessionWindow(21L, 42L)),
-                "value",
-                "oldValue");
+                new Record<>(
+                    new Windowed<>("key", new SessionWindow(21L, 42L)),
+                    new Change<>("value", "oldValue"),
+                    42L));
```

Comment:
Is this timestamp significant?

Reply:
not really, `SessionTupleForwarder` actually overrides timestamp:

```java
record.withValue(
                        new Change<>(
                            record.value().newValue,
                            sendOldValues ? record.value().oldValue : null))
                    .withTimestamp(record.key().window().end()));
```

Could make sense to move this up the stack.

Reply:
Huh, that's a weird thing for a "forwarder" to do, but not worth changing right now.